sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRegistryRecordedTestSanitizer.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Web;
+using Azure.Core.TestFramework;
+
+namespace Azure.Containers.ContainerRegistry.Tests
+{
+    public class ContainerRegistryRecordedTestSanitizer : RecordedTestSanitizer
+    {
+        public List<string> FormEncodedBodySanitizers { get; } = new List<string>();
+
+        public ContainerRegistryRecordedTestSanitizer()
+            : base()
+        {
+            JsonPathSanitizers.Add("$..access_token");
+            JsonPathSanitizers.Add("$..refresh_token");
+
+            FormEncodedBodySanitizers.Add("access_token");
+            FormEncodedBodySanitizers.Add("refresh_token");
+        }
+
+        public override string SanitizeTextBody(string contentType, string body)
+        {
+            string jsonSanitizedBody = base.SanitizeTextBody(contentType, body);
```

Comment:
@pakrym, would you review this use of sanitizer and see if it looks reasonable?  Feel free to send me back to docs on x-www-form-urlencoded if you think this method might have errors.

Reply:
Is this method being hit? I'm not sure if we recognize x-www-form-urlencoded as a "text" content type.

Reply:
It is being hit.  Should it not be?

Reply:
Meh, if it works it works :)